swagger.yaml

```diff
@@ -3250,6 +3250,21 @@ definitions:
         type: integer
         description: Required installation disk speed in ms
 
+  versioned-host-requirements:
+    type: object
+    properties:
+      version:
+        type: string
+        description: Version of the component for which requirements are defined
+      master:
+        x-go-name: MasterRequirements
+        $ref: '#/definitions/cluster-host-requirements-details'
+        description: Master node requirements
+      worker:
+        x-go-name: WorkerRequirements
+        $ref: '#/definitions/cluster-host-requirements-details'
```

Comment:
What's the difference between `cluster-host-requirements-details` and `host-requirements-role`?

Reply:
`host-requirements-role ` is used by the old `/host_requirements` endpoint and RAM is in GiB. The ` cluster-host-requirements-details` is used in the new endpoint and RAM is in MiB.

Reply:
I find it a little bit confusing

Reply:
`host-requirements` are supposed to be removed, as per https://issues.redhat.com/browse/MGMT-4220 - when the UI stops using it. 
WDYT about adding `deprecated: true` to `/host_requirements` for now instead?

Reply:
On the other hand... WDYT about temporarily adding `RamGiB` to the new model and removing the old one?

Reply:
Sorry, I meant to merge the models, not the APIs. While one uses GiB and the other Mib is really confusing. I think the usage should be aligned

Reply:
So anyway, there has to be a transition period where the model supports both units until UI switches over to MiB.

Reply:
I would like to make that change in a separate PR; is that OK with you?